x-pack/plugins/observability_solution/asset_manager/server/lib/entities/stop_and_delete_transform.ts

```diff
@@ -22,7 +22,7 @@ export async function stopAndDeleteHistoryTransform(
       () =>
         esClient.transform.stopTransform(
           { transform_id: historyTransformId, wait_for_completion: true, force: true },
-          { ignore: [409] }
+          { ignore: [409, 404] }
```

Comment:
what's the reason for adding this?

Reply:
consistency with [ingest pipeline deletion](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/asset_manager/server/lib/entities/delete_ingest_pipeline.ts#L22). also makes the definition uninstall code simpler. Throwing on 404 doesn't sound necessary since its not fatal